src/dropdownToggle/dropdownToggle.js

```diff
@@ -90,6 +90,16 @@ angular.module('mm.foundation.dropdownToggle', [ 'mm.foundation.position', 'mm.f
             }
           };
           $document.on('click', closeMenu);
+
+          angular.element($window).bind('resize', function() {
```

Comment:
No particular reason for `bind()`, changing.
Also, added cleanup on `$destroy` as suggested.